packages/twenty-front/src/modules/object-record/relation-picker/components/SingleEntitySelectMenuItemsWithSearch.tsx

```diff
@@ -16,6 +18,7 @@ export type SingleEntitySelectMenuItemsWithSearchProps = {
   relationObjectNameSingular: string;
   relationPickerScopeId?: string;
   selectedRelationRecordIds: string[];
+  clearOnOpen?: boolean;
```

Comment:
I think the behavior we want is:
- onSubmit or onCancel => reset to ''
- be able to programatically set a defaultValue in the Search (I don't think it's needed for now, we can let it for later)

Reply:
Remove this field